### PR TITLE
Soft Deleted Historical Property Entries not containing metadata information

### PR DESCRIPTION
**Pull request by sfeng88**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# v4.4.0
```
Comment (joeferner) [inline CHANGELOG.md]: 4.4.1

File: accumulo-iterators/src/main/java/org/vertexium/accumulo/iterator/model/PropertyColumnQualifier.java [hunk 2]
```diff
@@ -30,21 +30,34 @@ public String getDiscriminator(String visibilityString, long timestamp) {
         return getDiscriminator(getPropertyName(), getPropertyKey(), visibilityString, timestamp);
     }
 
+    public String getDiscriminator(String visibilityString) {
+        return getDiscriminatorWithoutTimestamp(getPropertyName(), getPropertyKey(), visibilityString);
+    }
+
     public static String getDiscriminator(String propertyName, String propertyKey, String visibilityString, long timestamp) {
+        String discriminatorWithoutTimestamp = getDiscriminatorWithoutTimestamp(propertyName, propertyKey, visibilityString);
```
Comment (joeferner) [inline accumulo-iterators/src/main/java/org/vertexium/accumulo/iterator/model/PropertyColumnQualifier.java]: Can you duplicate the logic from `getDiscriminatorWithoutTimestamp` here (ie leave `getDiscriminator` as it was). `getDiscriminator` is called a lot and any performance we can squeeze out the better.

**Comment by sfeng88** [comment]
@joeferner done

**Comment by joeferner** [comment]
@sfeng88 looks good. Once Travis finishes we can merge.

**Pull request merged**